src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -192,7 +191,16 @@ private void iterateBySimulations(
     int simulationsCount = simulations.size();
     for (int i = 0; i < simulationsCount; i++) {
       try {
-        executeGatling(jvmArgs, gatlingArgs(simulations.get(i)), testClasspath, toolchain);
+        String selectedSimulation = simulations.get(i);
+        String runDescription =
+            Optional.ofNullable(this.runDescription)
+                .orElse(ci ? null : Interactive.selectRunDescription());
+
+        List<String> gatlingArgs = gatlingArgs(selectedSimulation, runDescription);
+        if (!ci) {
+          System.out.println("Running simulation " + selectedSimulation + ".");
```

Comment:
I'm just wondering why this message isn't also displayed in the IC logs?

Reply:
We can, but not directly in sysout but in maven's logger in INFO level.